modules/ROOT/pages/rest-sdk/valueresolver.adoc

```diff
@@ -0,0 +1,250 @@
+= Define a Value Resolver
+ifndef::env-site,env-github[]
+include::../_attributes.adoc[]
+endif::[]
+
+// what is a value resolver?
```

Comment:
This feature is designed to provide the possible values of an operation’s parameter. 

Value resolvers will allow the connector to make a request to an endpoint and get the possible values from there, providing capabilities to extract the actual values from the server response.

Reply:
```suggestion
Value resolvers provide the possible values of an operation’s parameter.

Value resolvers also the connector to make a request to an endpoint and get the possible values from there, providing capabilities to extract the actual values from the server response.
```